index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Techsphere Conference</title>
+    <link rel="stylesheet" href="CSS/styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/vrw5nhb.css">
+    <script src="https://kit.fontawesome.com/5706a27a8e.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <main>
+        <section class="mobileMenu hide">
+            <div class="menuContainer">
+                <ul>
+                    <li> <a href="index.html" class="home">Home</a> </li>
+                    <li> <a href="about.html">About</a> </li>
+                    <li> <a href="#program">Program</a> </li>
+                    <li> <a href="#">Join</a> </li>
+                    <li> <a href="#">Sponsor</a> </li>
+                    <li> <a href="#">News</a> </li>                    
```

Comment:
- [OPTIONAL] I noticed you have separate navigations for mobile and desktop versions. it would be better if you had just one navigation and used CSS animations to move it around.